src/Tracker/index.js

```diff
@@ -61,10 +63,13 @@ const setUp = () => {
 
 const uninstall = () => {
   RavenTracker.uninstall()
+  woopra.dispose()
```

Comment:
It seems to be safe to do this, you probably have checked already, but here is the code it calls: https://github.com/onfido/js-client-tracker/blob/master/wpt.js#L1620

The cleaning of the global seems to be _safeish_, the for bit is obscure to me, but still an instance variable, so 👍

Reply:
Yeah, I did read that 🙂 tbh I just want _everything_ to do with whatever Woopra created to be destroyed.
I'd rather have errors be thrown in the console of "woopra tried to do a thing, but couldn't!" rather than Woopra trying to do a thing... and being able to 😨

Reply:
the issue is customers with their own woopra also having their woopra related data destroyed, we definitely want to avoid that. That's actually what I meant by _safeish_. `dispose` seems to destroy only instance related data, but if not, then we need to make the window manipulation safe before doing this.

Reply:
Ah, yes 🙂 I've just tested with multiple Woopra instances, and `.dispose()` of one instance is localised to just that instance